### PR TITLE
mount kube config

### PR DESCRIPTION
**Pull request by xiaolanz**
```release-note
none
```
issue istio/broker#15

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -96,57 +96,62 @@ presubmits:
...
+    always_run: true
+    rerun_command: "/test broker-presubmit"
+    trigger: "(?m)^/(retest|test (all|broker-presubmit))?,?(\\s+|$)"
+    branches:
+    - master
+    spec:
+      containers:
+      - image: gcr.io/istio-testing/prowbazel:0.2.1
+        args:
+        - "--repo=github.com/$(REPO_OWNER)/$(REPO_NAME)=$(PULL_REFS)"
```
Comment (sebastienvas) [inline prow/config.yaml:112]: Could you please add a timeout ?
--timeout=20, time in minute.

**Review by sebastienvas — COMMENTED** [review]
please add timeout

File: prow/config.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/config.yaml:112]: I am fixing the postsubmit in another PR.

**Comment by xiaolanz** [comment]
PTAL

**Review by sebastienvas — APPROVED** [review]

**Pull request merged**